### DISCUSSION TITLE
Per App launch in groups menu

### DISCUSSION THREAD

**Discussion by jonhall** [rxhanson/RectanglePro-Community#132] [Ideas]
You've a great app here. Just getting started with Groups. Any plans to include a per application launch in groups, it's on a group basis at the moment.

J

**Comment by rxhanson** [comment]
Sorry, looks like this one got lost in the shuffle. No plans to split out apps from groups at the moment, but would consider it with significant demand.

**Comment by jonhall** [comment]
Thanks, Ryan,


It’d be a cool feature.

Best wishes, Jon

On 2023-01-10, 03:22, Ryan Hanson wrote:


Sorry, looks like this one got lost in the shuffle. No plans to split out apps from groups at the moment, but would consider it with significant demand.
